### DISCUSSION TITLE
Getting SSEClient running

### DISCUSSION THREAD

**Discussion by uygix** [joqueka/bf4py#12] [Q&A]
Hi,

this library seems to be exactly what I have been looking for for quite a while, so thanks!
Unfortunately I can't get it running although I have already spent hours on trying to find the mistake. It seems to me that the url that is used for the request is not a valid SSE stream. It keeps on giving me the exception " 'SSEClient' object has no attribute 'events' " Do you have any experience with that?

For the first step I am actually only trying to extract the current price of ISIN XC000A0C4KJ2 . So if it would be possible for you to give me a hint on how to adjust the code in live_data and connector I would really be extremely grateful.

Thanks a lot and have a nice day!

**Comment by joqueka** [comment]
Hi,

your specified ISIN is of type commodity which uses another API endpoint which I haven't implemented yet due to missing demand. Now that I see someone needs this feature, I'll try to implement this as soon as possible but I'm quite busy at the moment.  

So there's nothing wrong with the connector as far as I can see from my quick check. Of course you can try to implement something on your own, based on the available functions, otherwise please be patient, since I'll need some days to integrate this.

Cheers!

**Comment by joqueka** [comment]
Hi again,

I was wrong. There's indeed another endpoint only for commodities but that is only to get a list of them grouped by category.

So far I can't reproduce your problem. You can use most of the regular instrument functions especially price_information() to get live data. But be sure to use it with parameter mic='ARIVA' in your case since that CO2 certificate is only available from that vendor. You can check that by using bf4py.general.instrument_information()

If you're still having troubles provide some more details.

Bests

**Reply by uygix** [reply]
Hi, 

thanks a lot for your help! Highly appreciated!!

Bests
